.github/workflows/check_extensions.yml

```
@@ -0,0 +1,51 @@
+      fail-fast: false
+      matrix:
+        include:
+          - name: default
+            valuesArgs: ""
+          - name: inline-secrets
+            valuesArgs: "-f ci/values-inline-secrets.yaml"
+          - name: existing-secrets
+            valuesArgs: "-f ci/values-existing-secrets.yaml"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
```

actionlint (1.7.8)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)